code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -224,6 +234,11 @@
 			else
 				if (!muzzled)
 					message = "giggles."
+					if(get_visible_gender() == MALE)
+						playsound(loc, "sound/voice/emotes/male_giggle[rand(1,2)].ogg", 50, 1)

```

Comment:
I suspect that the adding of sounds to human vocalization will land us quite firmly in the uncanny valley. Since it normally doesn't match what we expect our character to sound like.